packages/cli/src/lib/defaults/build-strategies/wasm/javascript/vm/VERSION

```diff
@@ -0,0 +1 @@
+0.1.6
```

Comment:
NIT: strange that we started this off at `0.1.6`

Reply:
Yeah, the image I published has no version tag. Just initted this with 0.0.1

Reply:
@namesty it looks like you might have accidentally published `0.1.6`, see here: https://hub.docker.com/r/polywrap/vm-base-js/tags

Not sure if we can "yank" the tag or not. Also let's start dev versioning at `0.1.0` like we do in our other software.